example/spec/parallel/discoverGranules/DiscoverGranulesDuplicateHandlingErrorSpec.js

```diff
@@ -1,202 +0,0 @@
-'use strict';
```

Comment:
may need to discuss this more but while working on this i ran into a failure that made me consider if this test case was still valid and i ended up deciding it wasn't since moving forward it should be a combination of collectionId and granuleId that determine aunthenticity of a granule. when i remove the test i realized that was the only test in the file so i deleted whole file.

Reply:
@nasamoduyebo and I talked about this briefly. This may be a good arch meeting topic (or similar). This PR is essentially saying that you can have duplicate `granuleIds`. The duplicate granule handling identifies a duplicate by the `granuleId`. I wonder if the solution is to effectively say duplicate granule handling is deprecated OR change the duplicate granule handling logic to support granuleId + collectionId.

Reply:
I think it does warrant further consideration - there's a legitimate case where duplication of granules occurs *for the same collection*, this is a use case DAACs/users currently have that we need to support.